lib/circuitbox/circuit_breaker.rb

```diff
@@ -52,15 +52,17 @@ def run!(run_options = {})
         logger.debug "[CIRCUIT] closed: querying #{service}"
 
         begin
-          t1 = Time.now
+          start_time = Time.now.to_i
```

Comment:
doing ```to_i``` will only give this to you in seconds so most things that happen under a second we'll either report a timing of 0 or 1.